Sources/Authenticator/Views/SignInView.swift

```diff
@@ -239,7 +240,7 @@ public struct SignInFooter: View {
     }
 
     public var body: some View {
-        HStack {
+        HStack(spacing: theme.components.authenticator.spacing.horizontal) {
```

Comment:
curious question, why isn't there a high-level spacing? i.e. `theme.components.spacing`? does this mean other components can have different spacing?

Reply:
> does this mean other components can have different spacing

Yes, that is what I had in mind. This "spacing" is meant to be the spacing that goes in `HStack`s and `VStack`s, and at least in the Authenticator I found it to be quite different between different components.